tasks/test_results_processor.py

```diff
@@ -68,8 +74,21 @@ def run_impl(
         testrun_dict_list = []
         upload_list = []
 
+        f = db_session.query(Flake).all()
```

Comment:
technically this is flakes

Reply:
oops, this is leftover from me debugging i think